src/applications/facility-locator/tests/helpers.unit.spec.js

```diff
@@ -213,4 +214,70 @@ describe('Validate ID Strings for Breadcrumb', () => {
 
     expect(result).to.eq(expected);
   });
+
+  it('isVADomain should return true if https://www.va.gov/pittsburgh-health-care/locations/beaver-county-va-clinic/ ', () => {
+    const result = isVADomain(
+      'https://www.va.gov/pittsburgh-health-care/locations/beaver-county-va-clinic/',
+    );
+    expect(result).to.eq(true);
+  });
+
+  it('isVADomain should return true if  https://www.va.gov/pittsburgh-health-care/locations/h-john-heinz-iii-department-of-veterans-affairs-medical-center/ ', () => {
+    const result = isVADomain(
+      'https://www.va.gov/pittsburgh-health-care/locations/h-john-heinz-iii-department-of-veterans-affairs-medical-center/',
+    );
+    expect(result).to.eq(true);
+  });
+
+  it('isVADomain should return true if  https://va.gov/testing ', () => {
+    const result = isVADomain('https://va.gov/testing');
+
+    expect(result).to.eq(true);
+  });
+
+  it('isVADomain should return true if  http://www.va.gov/testing ', () => {
+    const result = isVADomain('http://www.va.gov/testing');
+
+    expect(result).to.eq(true);
+  });
+
+  it('isVADomain should return true if  http://www.staging.va.gov/testing ', () => {
+    const result = isVADomain('http://www.staging.va.gov/testing');
```

Comment:
Does this work on [`dev.va.gov`](https://dev.va.gov)?

Reply:
we just need it to work on staging temporarily where we want to do the testing